zanataclient/zanatacmd.py

```diff
@@ -544,3 +545,26 @@ def delete_glossary(self, lang=None):
             self.log.error(str(e))
         else:
             self.log.info("Successfully delete the glossary terms on the server")
+
+    def get_project_translation_stats(self, project_id, project_version, min_doc_percent, lang_list, locale_map):
+        doc_locales_dict = {}
+        try:
+            server_return = self.zanata_resource.stats.get_project_stats(project_id, project_version)
+        except ZanataException, e:
+            self.log.error(str(e))
+        else:
+            percent_dict = Stats(server_return).trans_percent_dict
+            for doc, stat in percent_dict.items():
+                disqualify_locales = []
+                for locale, trans_percent in stat.items():
+                    if trans_percent < int(min_doc_percent):
+                        disqualify_locales.append(locale)
+                disqualify_locales = [alias for alias, locale in locale_map.items()
+                                      for lang in disqualify_locales if lang == locale]
+                if disqualify_locales:
+                    self.log.info('Translation file for document %s for locales [%s] are skipped due to '
+                                  'insufficient completed percentage' % (doc, ', '.join(map(str, disqualify_locales))))
```

Comment:
Could also include the min-doc-percent threshold in this message. "... are skipped because they are less than %s%% translated (--min-doc-percent setting)"